packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,108 @@
+
+Multiple disclosures can be used in a row for panels of unrelated content, at the client user's discretion and in accordance with component guidelines and product consistency.
+
+Avoid this:  
+![ ](spec-images/Accordion-15.png)
+
+If only one section (relative to the content) is needed, use a disclosure component.  
+![ ](spec-images/Accordion-16.png)
+
+### Tree & Navigation
+With a disclosure/accordion, the heading is a label for a group of content, not content in-and-of itself. With a tree, the content is the list items and all the list items represent data.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Parent/child |
+
+[ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/) recommends avoiding using a tree component for navigation because the `tree` role requires unecessary implementation of complex functionality that is not needed for navigation styled to look like a tree with expandable sections. However, the accordion is not necessarily the correct component to try and replace it with.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Hierarchical |
+| ???        | List items | Group labels |
+
+[**QUESTION:** Maybe a new navigation-specific component?]
+
+## Design
+### Interactions & States
+Header buttons are used to expand and collapse associated panels of content. The client user can specify which panels (if any) are open by default.
+
+![ ](spec-images/Accordion-1.png)
+
+Hovering over a header button indicates interactivity:  
+![ ](spec-images/Accordion-2.png)
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]  
+![ ](spec-images/Accordion-3.png)
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).  
+![ ](spec-images/Accordion-4.png)
+
+If a component in a panel is in an error state, the error state is visually indicated on the associated header button when closed (and maybe also be visually indicated when open).  
+![ ](spec-images/Accordion-5.png)
+
+### Additional Options
+Client users may need/want to have additional buttons and/or icons in the header. For example, an expand button:  
+![ ](spec-images/Accordion-6.png)
+
+### Nested Accordions
+Accordion panels can have one level of nested accordions. Nested accordions are always ghost style. [**QUESTION:** Can accordions be nested anywhere in the panel? Can there be multiple child accordions nested in an accordion?]  
+![ ](spec-images/Accordion-7.png)
+
+### Keyboard Interactions
```

Comment:
@leslieab, any reason to not include the other optional keyboard interactions ARIA outlines for the accordion (ARROWUP, ARROWDOWN, HOME, and END)?

Reply:
No, I'll add them.